proto/viam/app/v1/app.proto

```diff
@@ -441,6 +444,18 @@ message GetRobotRequest {
   string id = 1;
 }
 
+message GetRoverRentalRobotsRequest {}
+
+message RoverRentalRobot {
+  string robot_id = 1;
```

Comment:
we only need the robot_id because we will by default create a reservation with the "main" part of the robot.